frontend/src/app/main/ui/ds/notifications/actionable.scss

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) KALEIDOS INC
+
+@use "../_borders.scss" as *;
+
+.notification {
+  align-items: center;
+  background: var(--color-background-primary);
+  border-radius: $br-8;
+  border: $b-1 solid var(--color-background-quaternary);
+  display: flex;
```

Comment:
It's not bad, but usually `grid` simplifies some things since it adds an implicit `width` to the element (for things like cropping a line with an ellipsis), and offers more options, so I tend to favor it for the sake of maintanability.